src/dynamic_sharing_service/policy/dynamic.policy.js

```diff
@@ -13,11 +13,13 @@ module.exports = class DynamicPolicy {
    * @param {WantedAttribute[]} wantedAttributes - array of attributes to be requested.
    * @param {integer[]} wantedAuthTypes - auth types represents the authentication type to be used.
    * @param {boolean} wantedRememberMe
+   * @param {object} identityProfileRequirements
    */
   constructor(
     wantedAttributes,
     wantedAuthTypes,
-    wantedRememberMe = false
+    wantedRememberMe = false,
+    identityProfileRequirements
```

Comment:
Minor point, but having a required parameter after an optional parameter makes the default value of the optional parameter redundant

Reply:
I see you point, agreeing with the incorrect feeling it provides.
Yet, that parameter `identityProfileRequirements` is not mandatory, it is just new (and therefore we place it last), so should it be written as `identityProfileRequirements = undefined` in your opinion ?

Reply:
Oh I see. Yes, I prefer to be explicit about whether it's required or not. If it's not required I prefer `= null`.